### PR TITLE
Try improve wording of lifecycle tooltips

### PR DESCRIPTION
**Pull request by macobo**
## Changes

Related: https://github.com/PostHog/posthog/issues/7994

The new wording is definetly more accurate, but might also be confusing for users. Feel free to suggest improvements via comments!

### CODE REVIEW

File: frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx [hunk 1]
```diff
@@ -31,10 +31,10 @@ export function TrendTab({ view }: TrendTabProps): JSX.Element {
     const { groupsTaxonomicTypes } = useValues(groupsModel)
     const [isUsingFormulas, setIsUsingFormulas] = useState(filters.formula ? true : false)
     const lifecycles = [
-        { name: 'new', tooltip: 'Users that are new.' },
-        { name: 'resurrecting', tooltip: 'Users who were once active but became dormant, and are now active again.' },
-        { name: 'returning', tooltip: 'Users who consistently use the product.' },
-        { name: 'dormant', tooltip: 'Users who are inactive.' },
+        { name: 'new', tooltip: 'Users who signed up and did activity during period.' },
```
Comment (paolodamico) [inline frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx:34]:
```suggestion
        { name: 'new', tooltip: 'Users who both signed up and did the activity during the period .' },
```

Aside from suggestion above, how about interpolating the actual event name (e.g. "... and performed insight viewed.")?

File: frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx [hunk 2]
```diff
@@ -31,10 +31,10 @@ export function TrendTab({ view }: TrendTabProps): JSX.Element {
...
+        { name: 'returning', tooltip: 'Users who did activity both this and previous period.' },
```
Comment (paolodamico) [inline frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx:35]:
```suggestion
        { name: 'returning', tooltip: 'Users who did the activity in both this and the previous period. Users signed up before this period.' },
```

File: frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx [hunk 3]
```diff
@@ -31,10 +31,10 @@ export function TrendTab({ view }: TrendTabProps): JSX.Element {
...
+        { name: 'resurrecting', tooltip: 'Users who did activity this period but not the previous period.' },
```
Comment (paolodamico) [inline frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx:36]:
```suggestion
        { name: 'resurrecting', tooltip: 'Users who did the activity this period but did not do the activity on the previous period (i.e. were previously inactive for 1 or more periods).' },
```

File: frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx [hunk 4]
```diff
@@ -31,10 +31,10 @@ export function TrendTab({ view }: TrendTabProps): JSX.Element {
...
+        { name: 'dormant', tooltip: 'Users who did not do the activity this period but did it last period.' },
```
Comment (paolodamico) [inline frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx:37]:
```suggestion
        { name: 'dormant', tooltip: 'Users who went dormant on this period, i.e. users who did not do the activity this period but did the activity on the previous period.' },
```

**Review by paolodamico — CHANGES_REQUESTED** [review]
Some suggestions but great that we're making these more accurate

File: frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx (hunk 3 above)
Comment (macobo) [reply frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx:36]: This is inaccurate - user is only counted as dormant if they did it previous period and not this. If they did action t-2 they are not counted as dormant.

File: frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx (hunk 2 above)
Comment (macobo) [reply frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx:35]: I think referring to user sign ups outside new just makes everything confusing to follow - we'd need to mention it in resurrecting as well then.

Comment (paolodamico) [reply frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx:35]: With fresh eyes I also see another problem with mentioning sign ups. It's not exactly accurate. We take the first event for the user, which might not necessarily be a sign up (depending on how our users determine this). Amending my proposal.

```suggestion
        { name: 'returning', tooltip: 'Users who did the activity in both this and the previous period.' },
```

**Review by paolodamico — APPROVED** [review]

File: frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx [hunk 5]
```diff
@@ -31,10 +31,10 @@ export function TrendTab({ view }: TrendTabProps): JSX.Element {
     const { groupsTaxonomicTypes } = useValues(groupsModel)
     const [isUsingFormulas, setIsUsingFormulas] = useState(filters.formula ? true : false)
     const lifecycles = [
-        { name: 'new', tooltip: 'Users that are new.' },
-        { name: 'resurrecting', tooltip: 'Users who were once active but became dormant, and are now active again.' },
-        { name: 'returning', tooltip: 'Users who consistently use the product.' },
-        { name: 'dormant', tooltip: 'Users who are inactive.' },
+        { name: 'new', tooltip: 'Users who both signed up and did the activity during the period .' },
```
Comment (paolodamico) [inline frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx:34]:
```suggestion
        { name: 'new', tooltip: 'Users who were first seen on this period and did the activity during the period .' },
```

File: frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx [hunk 6]
```diff
@@ -31,10 +31,10 @@ export function TrendTab({ view }: TrendTabProps): JSX.Element {
...
+        { name: 'returning', tooltip: 'Users who did activity both this and previous period.' },
+        { name: 'resurrecting', tooltip: 'Users who did the activity this period but did not do the activity on the previous period (i.e. were previously inactive for 1 or more periods).' },
```
Comment (paolodamico) [inline frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx:36]:
```suggestion
        { name: 'resurrecting', tooltip: 'Users who did the activity this period but did not do the activity on the previous period (i.e. were inactive for 1 or more periods).' },
```

**Pull request merged**